WordPress/src/main/java/org/wordpress/android/ui/blaze/BlazeActionEvent.kt

```diff
@@ -0,0 +1,6 @@
+package org.wordpress.android.ui.blaze
+
+sealed class BlazeActionEvent {
+    object FinishActivity : BlazeActionEvent()
```

Comment:
Hey @zwarm 👋🏼 

🗒️  I have added some logic for finishing the activity as well on close/back click of the overlay, I can update my logic to use this event after these changes land in the `trunk`. So you might see that logic in [PR](https://github.com/wordpress-mobile/WordPress-Android/pull/18018), Please feel free to ignore that during the review. 

Having a separate event for Action event 👏🏼 . Nice.

Reply:
@AjeshRPai Thanks for the heads up. After Done is clicked, I am passing back the FinishActivity which is being handled in the BlazeWebViewFragment. 🎈